Neos.ContentGraph.DoctrineDbalAdapter/Tests/Behavior/Features/Bootstrap/ProjectionIntegrityViolationDetectionTrait.php

```diff
@@ -54,9 +54,9 @@ abstract private function getObject(string $className): object;
 
     protected function getTableNamePrefix(): string
     {
-        return DoctrineDbalContentGraphProjectionFactory::graphProjectionTableNamePrefix(
+        return ContentGraphTableNames::create(
             $this->currentContentRepository->id
-        );
+        )->tableNamePrefix;
```

Comment:
(why) is this method still needed if we don't work with the table name prefixes any longer?

Reply:
it is not if we fully rewrite this.